src/users/users.controller.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { UsersController } from './users.controller';
+import { UsersService } from './users.service';
+import * as consts from '../test.constants';
+import { CreateUserDto } from './dto/create-user.dto';
+import { UserDto } from './dto/user.dto';
+import { UpdateUserDto } from './dto/update-user.dto';
+import { DeleteUserDto } from './dto/delete-user.dto';
+
+// Test suite for the UsersController
+describe("UsersController Unit Tests", () => {
```

Comment:
Throughout these controller tests it would be good to see some negative testing in addition to the positive tests that you've started on. For example - test what happens when an ID is provided that is not found. Test what happens when missing required parameters during create, etc.